src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationPerformanceTest.kt

```diff
@@ -0,0 +1,94 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.alerts
```

Comment:
Over the top test that we can delete in the future, I just wanted to reproduce the exact scenario before applying fix